changelog.md

```diff
@@ -1,65 +1,68 @@
+* Fixed bug in [`lk.track_lines()`](https://lumicks-pylake.readthedocs.io/en/v1.0.0/_api/lumicks.pylake.track_lines.html) where one extra line was returned rather than the number requested through the parameter `max_lines`.
+* Fixed regression that caused [`ImageStack`](https://lumicks-pylake.readthedocs.io/en/v1.0.0/_api/lumicks.pylake.ImageStack.html) to incorrectly store the stack name (resulting in an extra `('` in front of the name in plots). This regression was introduced in `v0.13.0` when enabling multi-file support. See [`ImageStack`](https://lumicks-pylake.readthedocs.io/en/v1.0.0/_api/lumicks.pylake.ImageStack.html) for more information.
 
 #### Other changes
 
-* Removed all deprecated methods and properties from `Kymo`, `Scan`, `PointScan`, and `ImageStack`.
+* Removed all deprecated methods and properties from [`Kymo`](https://lumicks-pylake.readthedocs.io/en/v1.0.0/_api/lumicks.pylake.kymo.Kymo.html), [`Scan`](https://lumicks-pylake.readthedocs.io/en/v1.0.0/_api/lumicks.pylake.scan.Scan.html), [`PointScan`](https://lumicks-pylake.readthedocs.io/en/v1.0.0/_api/lumicks.pylake.point_scan.PointScan.html), and [`ImageStack`](https://lumicks-pylake.readthedocs.io/en/v1.0.0/_api/lumicks.pylake.ImageStack.html).
 * Removed deprecated functions `filter_lines()`, `refine_lines_centroid()` and `refine_lines_gaussian()`.
 * Removed deprecated property `lines` and method `save_lines` from `KymoWidgetGreedy`.
-* Removed deprecated argument `exclude` from `Kymo.line_timestamp_ranges()` and `Scan.frame_timestamp_ranges()`.
-* Removed deprecated argument `line_width` from `lk.track_greedy()`.
+* Removed deprecated argument `exclude` from [`Kymo.line_timestamp_ranges()`](https://lumicks-pylake.readthedocs.io/en/v1.0.0/_api/lumicks.pylake.kymo.Kymo.html#lumicks.pylake.kymo.Kymo.line_timestamp_ranges) and [`Scan.frame_timestamp_ranges()`](https://lumicks-pylake.readthedocs.io/en/v1.0.0/_api/lumicks.pylake.scan.Scan.html#lumicks.pylake.scan.Scan.frame_timestamp_ranges). Use the argument `include_dead_time` instead.
+* Removed deprecated argument `line_width` from [`lk.track_greedy()`](https://lumicks-pylake.readthedocs.io/en/v1.0.0/_api/lumicks.pylake.track_greedy.html).
```

Comment:
I put these as `Other changes` originally but I wonder if they should be listed under `Breaking changes`. Although I guess because we went through a deprecation pathway they're not technically breaking? thoughts?

Reply:
They are. I'll move them.